cmd/sysl/cmd_datamodelview.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"strings"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	sysl "github.com/anz-bank/sysl/pkg/sysl"
+	"github.com/sirupsen/logrus"
+)
+
+func GenerateDataModelsView(datagenParams *CmdContextParamDatagen,
```

Comment:
This should be moved into a package, not in main

Reply:
I checked code and found it had more dependences on package main like seqs.go, fmtParser.go.
So I think it is better to put it in main package still as our discussion. And I moved test data to /tests/datamodel to make sure it is clear.